File: lectures/week-1/exercises/meow.c
#include <stdlib.h>

int main(void)
{
    int manyTimesSayMeow;

    printf("How many times do you want me to say meow? ");
    scanf("%d", &manyTimesSayMeow);

    for (int i = 0; i < manyTimesSayMeow; i++)
    {
        printf("Meow\n");
    }
}

File: lectures/week-6/exercises/meow.py
def meow(number: int):
    for i in range(number):
        print(f"meow {i + 1}")

def main():
    number = 0

    while True:
        try:  
            number = int(input("How many times do you want to meow? "))          
            if number < 0:
                print("Please enter a positive number.")
            else:
                break
        except ValueError:
            print("Please enter a valid number.")
    
    meow(number)

if __name__ == "__main__":
    main()